demos/shadow/shadow_demo_main/shadow_demo_main.c

```diff
@@ -640,17 +657,17 @@ static void eventCallback( MQTTContext_t * pMqttContext,
  * Device Shadow library to assemble strings for the MQTT topics defined
  * by AWS IoT Device Shadow. It uses these macros for topics to subscribe
  * to:
- * - SHADOW_TOPIC_STRING_UPDATE_DELTA for "$aws/things/thingName/shadow/update/delta"
- * - SHADOW_TOPIC_STRING_UPDATE_ACCEPTED for "$aws/things/thingName/shadow/update/accepted"
- * - SHADOW_TOPIC_STRING_UPDATE_REJECTED for "$aws/things/thingName/shadow/update/rejected"
+ * - SHADOW_TOPIC_STRING_UPDATE_DELTA for "$aws/things/thingName/shadow[/name/shadowname/]/update/delta"
```

Comment:
Maybe mention in the paragraph above the optional nature of the bracket strings?

Reply:
Done.